README.md

```diff
@@ -44,6 +44,7 @@ Error: duplicated attrname `a`
 | `--variable-lookup [<VARIABLE_LOOKUP>]` | Enable variable lookup analysis [default: true] [possible values: true, false] |
 | `-h, --help`                            | Print help                                                                     |
 | `-V, --version`                         | Print version                                                                  |
+| `-i, --ignore <IDs>`                    | Ignore diagnostics with specific ids                                           |
```

Comment:
Is this the same as how it appears in `--help`?

I think the documentation could be clearer w.r.t. specifying multiple IDs:

- Should they be a delimited list?
- Simply pass `--ignore` multiple times?
- If delimited, what's the deliminator?
- If passed multiple times, I think `<IDs>` (plural) is confusing.